cmd/tether/attach.go

```diff
@@ -464,6 +483,13 @@ func (t *attachServerSSH) globalMux(reqchan <-chan *ssh.Request) {
 			}
 			msg := msgs.ContainersMsg{IDs: keys}
 			payload = msg.Marshal()
+
+			// unblock ^ (above)
+			pendingFn = func() {
+				once.Do(func() {
+					close(t.askedAndAnswered)
```

Comment:
We need container-ids in the portlayer before unblocking the process which is what this request responds (per attach operation)